src/app/shared/data-table/data-table.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="data-table-wrapper">
   <div class="data-table-header">
-    <button mat-raised-button class="data-table-add-button">Basic</button>
+    <button mat-raised-button class="data-table-add-button"mat-button color="primary" (click)="addButtonClicked()" >Add</button>
```

Comment:
@yosefmare 
Why did you add `mat-button`?

Reply:
fixed